usrsctplib/netinet/sctp_pcb.c

```diff
@@ -4151,10 +4149,10 @@ sctp_inpcb_free(struct sctp_inpcb *inp, int immediate, int from)
 	if (SCTP_ASOC_CREATE_LOCK_CONTENDED(inp))
 		being_refed++;
 
+	// NOTE: 0 refcount also means no timers are referencing us.
 	if ((inp->refcount) ||
 	    (being_refed) ||
 	    (inp->sctp_flags & SCTP_PCB_FLAGS_CLOSE_IP)) {
-		(void)SCTP_OS_TIMER_STOP(&inp->sctp_ep.signature_change.timer);
```

Comment:
This was already stopped above, so I don't think it needs to be called again?

Reply:
Note: the place where it stopped is here:

https://github.com/sctplab/usrsctp/blob/0f1de1e6896523ac790ca0880fd6c91dc8a3d7cb/usrsctplib/netinet/sctp_pcb.c#L3932-L3933